.github/workflows/pr-test.yml

```diff
@@ -24,6 +24,44 @@ jobs:
         with:
           tools-deps: '1.10.1.469'
 
-      - name: Unit Test
+      # Get secret from vault
+      # Lifted from https://github.com/broadinstitute/gotc-deploy/'s actions
+      - name: Authenticate to Gcloud
+        id: gcloud
+        env:
+          ROLE_ID: ${{ secrets.ROLE_ID }}
+          SECRET_ID: ${{ secrets.SECRET_ID }}
+          VAULT_ADDR: ${{ secrets.VAULT_ADDR }}
+          SA_KEY_PATH: ${{ secrets.SA_KEY_PATH }}
```

Comment:
generally we don't bother with protecting our paths that we need.  it just makes things difficult to manage and adds a level of indirection that IMO makes things more difficult to read and maintain.  and really the paths themselves are not secret - the contents are.

Reply:
Fair enough--fixed with [this commit](https://github.com/broadinstitute/push-to-cloud-service/commit/670596a034c9cf75829ef206b1fb3708945b1f6d)